changelog.txt

```diff
@@ -17,6 +17,7 @@ V1.XX.X
 [QMS-116] Filter edit field lost
 [QMS-118] Too much whitespace in extended search help window
 [QMS-119] Better handling of disabled and archived geocaches
+[QMS-126] Wrong results of status search
```

Comment:
I know this is the ticket title but it's not really descriptive. Can you please change the log entry to:

Wrong results searching for geocache status